src/components/Chips/Chips.tsx

```diff
@@ -184,6 +177,8 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
         }
       : {
           ref: mergedRef,
+          onClick: onClickCallback,
```

Comment:
If someone using onclick/onmousedown as hack and not want to treat chip as a button (like we do in drop down multi state) we still want that the onClick/onMouseDown will work as usual (but without hover appearance changes or aria attributes)